pkg/app/src/components/Account/modules/IdentitySection/form.tsx

```diff
@@ -100,7 +96,7 @@ export function IdentityForm() {
 									render={({ field: { onChange, value }, formState: { errors } }) => (
 										<TextField
 											fullWidth
-											label="Display Name"
+											label={t('label:display_name')}
 											placeholder="QDozer"
```

Comment:
Translation here as well

Reply:
@PAkhidue placeHolder is an example of a display name, translation is done here!